src/pages/health-declaration/health-declaration.scss

```diff
@@ -26,3 +26,24 @@ page-health-declaration {
     }
   }
 }
+.confirm-declaration-modal {
+  .alert-head:before {
+    display: inline-block;
+    font-size: 68px;
+    content: '\f276';
+    display: inline-block;
+    font-family: "Ionicons";
+    -moz-osx-font-smoothing: grayscale;
+    -webkit-font-smoothing: antialiased;
+    font-style: normal;
+    font-variant: normal;
+    font-weight: normal;
+    line-height: 1;
+    text-rendering: auto;
+    text-transform: none;
+  }
```

Comment:
is this not already in the `main.css` file?

Reply:
I don't believe so.